### PR TITLE
build: update build env to use go 1.16

### PR DESCRIPTION
**Pull request by troian**

### CODE REVIEW

**Review by hydrogen18 — APPROVED** [review]

**Review by hydrogen18 — APPROVED** [review]

File: .github/workflows/k8s-integration.yaml [hunk 1]
```diff
@@ -17,9 +17,15 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - shell: bash
+        run: |
+          echo "loading env from .makerc"
+          while IFS='' read -r LINE || [ -n "${LINE}" ]; do
+            echo "$LINE" >> $GITHUB_ENV;
+          done < .makerc
```
Comment (boz) [inline .github/workflows/k8s-integration.yaml:25]: `cat .makerc >> "$GITHUB_ENV`?

File: tools.go [hunk 2]
```diff
@@ -4,9 +4,6 @@ package tools
 
 //nolint
 import (
-	_ "github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger"
-	_ "github.com/regen-network/cosmos-proto/protoc-gen-gocosmos"
-	_ "github.com/vektra/mockery"
 	_ "golang.org/x/tools/cmd/stringer"
```
Comment (boz) [inline tools.go:7]: I think we don't use stringer at all anymore.

**Review by boz — APPROVED** [review]
:+1:

File: .github/workflows/k8s-integration.yaml (hunk 1 above)
Comment (troian) [reply .github/workflows/k8s-integration.yaml]: that won't work sadly.
changed to use `c-py/action-dotenv-to-setenv@v3` instead

**Pull request merged**